spec/bundler/shared_helpers_spec.rb

```diff
@@ -261,6 +261,30 @@
       subject.set_bundle_environment
     end
 
+    it "exits if bundle path contains the path seperator" do
+      stub_const("File::PATH_SEPARATOR", ":".freeze)
+      allow(Bundler).to receive(:bundle_path) { "so:me/dir/bin" }
+      begin
+        expect(subject.send(:validate_bundle_path)).to raise_exception(SystemExit).
+          and output("WARNING: Your bundle path contains a ':', which is the" \
+                     " path separator for your system. Please change your" \
+                     " bundle path path to not include ':'.")
+      rescue SystemExit => e
+        expect(e.status).to eq(1)
+      end
+
+      stub_const("File::PATH_SEPARATOR", "^".freeze)
```

Comment:
I only added it to test that the error message changes when the path separator is different than ```:```. I suppose it's redundant. I can remove it.